### ISSUE TITLE
Add support for '3BFF1800008131FE45006B0505912001F101434E5310318038' token

### ISSUE THREAD

**Issue by defacto64** [PeculiarVentures/fortify#529]
Reader name: BIT4ID miniLector EVO 0
ATR: 3BFF1800008131FE45006B0505912001F101434E5310318038

```
{
   "cards": [{
   	"atr": "3BFF1800008131FE45006B0505912001F101434E5310318038",
   	"name": "IDEMIA ID-One CNS V2 on Cosmo V9.1",
   	"driver": "DBADF7126B4E3656FA5E8CF2AE8D712DE953BC98"
   }],
}
```

[Smart card ATR parsing 3BFF1800008131FE45006B0505912001F101434E5310318038](https://smartcard-atr.apdu.fr/parse?ATR=3BFF1800008131FE45006B0505912001F101434E5310318038)

**Comment by defacto64** [comment]
Please add this ATR which identifies a Cosmo 9.1-based TS-CNS smartcard, to be driven by the already-known driver DBADF7126B4E3656FA5E8CF2AE8D712DE953BC98.
Our tests are okay.

**Comment by microshine** [comment]
We've published the new version of `cards@1.1.11`. Please restart your Fortify application to get the latest version

**Comment by defacto64** [comment]
It works fine, thank you Stepan.
